app/views/user_verification/steps/_details.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class="govuk-label govuk-label--l">Register for the FPO Commodity Code Identification Tool API</h1>
+
+<p class="govuk-body">
+To allow you to use the FPO Commodity Code Identification Tool API we need to check that your organisation is UK Carrier Scheme (UKC) registered.
+</p>
+
+<p class="govuk-body">
+Enter the following information about your organisation. This will be used to verify your account.
+</p>
+
+<%= f.govuk_text_field :organisation_name,
+  label: { text: "What is the name of your organisation?", class: "govuk-label--m" },
+  hint: { text: "This should be the name of the organisation you are accessing the Commodity Code Tool on behalf of." } %>
+
+<%= f.govuk_text_field :eori_number,
+  label: { text: "What is your EORI number?", class: "govuk-label--m" },
+  hint: { text: "Your organisation will have an Economic Operators Registration and Identification (EORI) number." } %>
+
+<%= f.govuk_text_field :ukacs_reference,
+  label: { text: "What is your UK Carrier Scheme (UKC) reference number?", class: "govuk-label--m" },
+  hint: { text: "Your organisation would have received a unique reference number when signing up for the UK Authorised Carrier Scheme." } %>
+
+<%= f.govuk_email_field :email_address,
+  label: { text: "Your work email address", class: "govuk-label--m" },
+  hint: { text: "We will only use your email address to send essential updates regarding the Commodity Code Identification Tool and your verification request." },
+  autocomplete: "email",
+  spellcheck: false %>
```

Comment:
Could put the label and hint text in the translations file?
Also, do you need to specify the class for the labels? There is a size attribute that might do what you want - e.g. { size: 'm' }

Reply:
I'd actually be happy to go with the defaults for the class and just follow the convention.

Reply:
I could put these in the translation file yeah.